pkg/nimble/contrib/nimble_riot.c

```diff
@@ -105,6 +105,11 @@ void nimble_riot_init(void)
     int res;
     (void)res;
 
+#if !IS_USED(MODULE_MYNEWT_CORE) && IS_ACTIVE(NIMBLE_CFG_CONTROLLER)
+    hal_timer_init(5, NULL);
+    os_cputime_init(32768);
```

Comment:
What are those magic numbers?

Reply:
The 5th timer is NRF_RTC0 in nimble/mynewt. And the frequency is the maximum possible frequency, but there are actually defines for this I'll address.

Reply:
FIxed the magic values and added a comment on mynewt-core and mynewt-nimble for this so its a bit less obscure